manage-engine-service-desk-plus-msp/info.json

```diff
@@ -0,0 +1,1944 @@
+              "profile_pic": {
+                "content-url": "",
+                "name": ""
+              },
+              "is_vipuser": "",
+              "org_user_status": "",
+              "id": "",
+              "department": {
+                "site": {
+                  "name": "",
+                  "id": ""
+                },
+                "name": "",
+                "id": ""
+              },
+              "account": {
+                "name": "",
+                "id": ""
+              }
+            },
+            "ciid": "",
+            "purchase_approval_limit": "",
+            "login_name": "",
+            "requester_allowed_to_view": "",
+            "phone": "",
+            "sip_user": "",
+            "employee_id": "",
+            "domain": "",
+            "all_domain": "",
+            "name": "",
+            "enable_telephony": "",
+            "account": {
+              "country": "",
+              "inactive": "",
+              "has_attachments": "",
+              "name": "",
+              "id": "",
+              "ciid": ""
+            },
+            "status": ""
+          }
+        ]
+      },
+      "enabled": true,
+      "parameters": [
+        {
+          "title": "User ID",
+          "name": "id",
+          "visible": true,
+          "required": false,
+          "editable": true,
+          "type": "text",
+          "tooltip": "Specify the ID of the user whose details you want to retrieve from ServiceDesk Plus MSP.",
+          "description": "Specify the ID of the user whose details you want to retrieve from ServiceDesk Plus MSP."
+        },
+        {
+          "title": "Name",
+          "name": "name",
+          "visible": true,
+          "required": false,
+          "editable": true,
+          "type": "text",
+          "tooltip": "Specify the name of the user whose details you want to retrieve from ServiceDesk Plus MSP.",
+          "description": "Specify the name of the user whose details you want to retrieve from ServiceDesk Plus MSP."
+        },
+        {
+          "title": "Employee ID",
+          "name": "employee_id",
+          "visible": true,
+          "required": false,
+          "editable": true,
+          "type": "text",
+          "tooltip": "Specify the employee ID of the user whose details you want to retrieve from ServiceDesk Plus MSP.",
+          "description": "Specify the employee ID of the user whose details you want to retrieve from ServiceDesk Plus MSP."
+        },
+        {
+          "title": "Email ID",
+          "name": "email_id",
+          "visible": true,
+          "required": false,
+          "editable": true,
+          "type": "text",
+          "tooltip": "Specify the email ID of the user whose details you want to retrieve from ServiceDesk Plus MSP.",
+          "description": "Specify the email ID of the user whose details you want to retrieve from ServiceDesk Plus MSP.",
+          "options": [
```

Comment:
why these options are given ?

Reply:
It should have value true,false or empty